SenecaAurelius/liftoff-project
# 10 child experiment

I've refactored a TON of code here—Parent and Child are now unique objects (that extend AbstractEntity), and each one is associated with a separate ParentUser/ChildUser object that governs the authentication piece.

Testing Steps:
1. download and run locally
2. You might need to Drop all tables in MYSQL because there are all sorts of new things
3. Create a parent user
4. Navigate to `localhost:8080/crew`
5. Add crew member at `localhost:8080/crew/add`
6. Add another crew member
7. Logout
8. Create another user
9. Navigate to `localhost:8080/crew` and make sure there are no crew members
10. Add new crew members
